lib/ManualImageCropEditorWindow.php

```diff
@@ -94,6 +94,15 @@ public function renderWindow() {
 				$src_file = str_replace($uploadsDir['baseurl'], $uploadsDir['basedir'], $src_file_url[0]);
 				$sizes = getimagesize($src_file);
 
+				$original[0] = $sizes[0];
+				$original[1] = $sizes[1];
+
+				if ($width > $sizes[0]) {
```

Comment:
Hi Odie,
thanks for your PR.
Can you explain what's going on here?
(moze byc po polsku :) )